bugsnag-plugin-android-exitinfo/src/main/proto/tombstone.proto

```diff
@@ -0,0 +1,170 @@
+//
+// Protobuf definition for Android tombstones.
+//
+// An app can get hold of these for any `REASON_CRASH_NATIVE` instance of
+// `android.app.ApplicationExitInfo`.
+//
+// https://developer.android.com/reference/android/app/ApplicationExitInfo#getTraceInputStream()
+//
+syntax = "proto3";
+option java_package = "com.android.server.os";
```

Comment:
I think it might be worth renaming this package to avoid collisions:
```suggestion
option java_package = "com.bugsnag.android.repackaged.server.os";
```